README.md

```diff
@@ -0,0 +1,12 @@
+# README - copy on write
+
+Docker image to maintain a duplicated directory structure. Works based on filesystem events. Copies a file/directory from a tracked source directory when it is created or moved to the source directory.
+Works based on regex substitution. See [replacements](localdev/replacements.sed) for an example mapping.
+
+On startup existing files/directories in SOURCE_ROOT are mapped and copied to target if mapped.
+
+
+See `localdev` for usage
```

Comment:
pls linke to the according file the user shoud look at

add info about how to run the tests